pkg/controller/clusterpool/collections.go

```diff
@@ -636,34 +648,39 @@ func getAllCustomizationsForPool(c client.Client, pool *hivev1.ClusterPool, logg
 		cloud:      make([]*hivev1.ClusterDeploymentCustomization, 0),
 		syntax:     make([]*hivev1.ClusterDeploymentCustomization, 0),
```

Comment:
The only use case I can think of right away is the message in InventoryValid condition. Maybe in the future it would be more useful.. I don't mind adding it, but maybe change it to `pendining` and `installed` for successful installations (to match CD spec.Installed). What do you think?

Reply:
Only if it's going to be useful somewhere. The more lists we add, the more complicated things become, especially when a CDC changes state.

Reply:
I prefer the unix philosophy of no output on successful run, so I will not add the `pending` and `installed` list at this point.

Reply:
Agree. YAGNI.